diff --git a/share/org.videolan.vlc.appdata.xml.in.in b/share/org.videolan.vlc.appdata.xml.in.in
index 62bd7ee443..ec237dc565 100644
--- a/share/org.videolan.vlc.appdata.xml.in.in
+++ b/share/org.videolan.vlc.appdata.xml.in.in
@@ -18,7 +18,7 @@
     <url type="bugtracker">https://trac.videolan.org/vlc/</url>
     <url type="donation">https://www.videolan.org/contribute.html</url>
     <releases>
-        <release version="@VERSION@" date="@VERSION_DATE@" />
+        <release version="@VERSION@" />
     </releases>
     <provides>
         <library>libvlc.so.5</library>
@@ -29,13 +29,13 @@
     <update_contact>vlc-devel -- videolan.org</update_contact>
     <screenshots>
         <screenshot type="default">
-            <image>https://images.videolan.org/vlc/screenshots/2.0.0/vlc-2.0-poney.jpg</image>
+            <image>http://images.videolan.org/vlc/screenshots/2.0.0/vlc-2.0-poney.jpg</image>
         </screenshot>
         <screenshot>
-            <image>https://images.videolan.org/vlc/screenshots/2.0.0/vlc-2.0-gnome3-open.jpg</image>
+            <image>http://images.videolan.org/vlc/screenshots/2.0.0/vlc-2.0-gnome3-open.jpg</image>
         </screenshot>
         <screenshot>
-            <image>https://images.videolan.org/vlc/screenshots/2.0.0/vlc-2.0-gnome3-debian.jpg</image>
+            <image>http://images.videolan.org/vlc/screenshots/2.0.0/vlc-2.0-gnome3-debian.jpg</image>
         </screenshot>
     </screenshots>
     <content_rating type="oars-1.1" />
