diff --git a/share/Makefile.am b/share/Makefile.am
index a6c0ac1083..7afbfa5dd6 100644
--- a/share/Makefile.am
+++ b/share/Makefile.am
@@ -5,17 +5,16 @@ EXTRA_DIST =
 desktopdir = $(datadir)/applications
 if !HAVE_WIN32
 if !HAVE_DARWIN
-desktop_DATA = vlc.desktop vlc-openbd.desktop vlc-opendvd.desktop vlc-openvcd.desktop vlc-opencda.desktop
+desktop_DATA = vlc.desktop
 appdatadir = $(datarootdir)/metainfo
 appdata_DATA = org.videolan.vlc.appdata.xml
 endif
 endif
 
 EXTRA_DIST += vlc.desktop.in vlc.desktop.mimetypes
-EXTRA_DIST += vlc-openbd.desktop.in vlc-opendvd.desktop.in vlc-openvcd.desktop.in vlc-opencda.desktop.in
 CLEANFILES += $(desktop_DATA) $(appdata_DATA)
 
-%.desktop: %.desktop.in $(top_builddir)/config.status
+vlc.desktop: vlc.desktop.in $(top_builddir)/config.status
 	$(AM_V_GEN)mimetypes="$$(sed 's/\s*#.*$$//g' $(srcdir)/vlc.desktop.mimetypes | $(EGREP) -v '^$$' | tr "\n" ';')"; \
 	sed \
 		-e 's,\@bindir\@,$(bindir),g' \
@@ -154,6 +153,7 @@ nobase_vlclib_DATA += \
 	lua/playlist/soundcloud.luac \
 	lua/playlist/vimeo.luac \
 	lua/playlist/vocaroo.luac \
+	lua/playlist/youtube.luac \
 	lua/playlist/twitch.luac \
 	lua/sd/icecast.luac \
 	lua/sd/jamendo.luac \
@@ -173,6 +173,7 @@ nobase_doc_DATA = \
 	lua/meta/reader/filename.lua \
 	lua/playlist/README.txt \
 	lua/playlist/liveleak.lua \
+	lua/playlist/youtube.lua \
 	lua/sd/README.txt \
 	lua/sd/icecast.lua \
 	lua/sd/icast.lua \
@@ -290,6 +291,7 @@ EXTRA_DIST += \
 	lua/playlist/soundcloud.lua \
 	lua/playlist/vimeo.lua \
 	lua/playlist/vocaroo.lua \
+	lua/playlist/youtube.lua \
 	lua/playlist/twitch.lua \
 	lua/playlist/zapiks.lua \
 	lua/sd/README.txt \
diff --git a/share/vlc.desktop.mimetypes b/share/vlc.desktop.mimetypes
index d521c6c34f..0d866f7da8 100644
--- a/share/vlc.desktop.mimetypes
+++ b/share/vlc.desktop.mimetypes
@@ -119,6 +119,10 @@ x-scheme-handler/icyx   # Icecast
 
 # Linux desktop environment hooks for ISOs etc.
 application/x-cd-image
+x-content/video-vcd
+x-content/video-svcd
+x-content/video-dvd
+x-content/audio-cdda
 x-content/audio-player
 
 # Playlists / text/xml list with URLs
